docs/integration_quality_scale_index.md

```diff
@@ -21,7 +21,7 @@ This integration is able to cope when things go wrong. It will not print any exc
 
 - Connection/configuration is handled via a component.
 - Set an appropriate `SCAN_INTERVAL` (if a polling integration)
-- Raise `PlatformNotReady` if unable to connect during platform setup (if appropriate)
+- Raise `PlatformNotReady` or `ConfigEntryNotReady` if unable to connect during platform setup (if appropriate)
```

Comment:
`ConfigEntryNotReady` is not used for platform setup.

I think this belongs in the Gold level section, as that is the section that introduces config entries, to begin with.

Reply:
Ok, I'll move this